### PR TITLE
Add ‘no visited state' link variant

### LINKED ISSUE #297: Add a link style for links with no visited state
There are some cases where it’s not appropriate to distinguish between visited and non-visited links.

An example is navigation links to pages with dynamic content, like dashboards in admin systems. The content on the page is changing all the time, so the fact that you’ve visited it before is not that important.

### PR DESCRIPTION
**Pull request by 36degrees**
Add an ‘no visited state' link variant which can be used where it is not helpful to distinguish between visited and non-visited links.

For example, navigation links to pages with dynamic content like admin dashboards. The content on the page is changing all the time, so the fact that you’ve visited it before is not important.

Closes #297

https://trello.com/c/BF3hYuXJ/622-add-link-variant-for-links-with-no-visited-state

### CODE REVIEW

File: src/globals/scss/core/_links.scss [hunk 1]
```diff
@@ -63,4 +63,18 @@
...
+  // 'Unvisited' link variant
+  //
+  // Used in cases where it is not helpful to distinguish between visited and
+  // non-visited links.
+  //
+  // For example, navigation links to pages with dynamic content like admin
+  // dashboards. The content on the page is changing all the time, so the fact
+  // that you’ve visited it before is not important.
+  .govuk-link--unvisited {
+    &:visited {
```
Comment (nickcolley) [inline src/globals/scss/core/_links.scss]: Can we leave this unnested?

Comment (36degrees) [reply src/globals/scss/core/_links.scss]: We can, though I think there is some value in being consistent with the way the rest of the file is written, making it easier to scan. I also personally think it'd be easy to miss if it was on the same line as the class name.

Comment (nickcolley) [reply src/globals/scss/core/_links.scss]: Ah sorry, I didn't see the rest of the file in this diff.

Sounds good to me 👍

**Comment by 36degrees** [comment]
@nickcolley have changed to `govuk-link--no-visited-state` which I think makes more sense having talked it through with Tim. Would you mind just re-checking it?

**Comment by nickcolley** [comment]
@36degrees makes sense to me, a bit verbose but hopefully very clear 👍

**Comment by 36degrees** [comment]
Mind approving again? I dismissed your previous review.

**Comment by 36degrees** [comment]
@nickcolley Good shout. Done 👍

**Pull request merged**